red_queen/games/applications/__init__.py

```diff
@@ -32,7 +33,9 @@
 ]
 
 
-def run_qiskit_circuit(benchmark, circuit, backend, optimization_level, shots, expected_counts):
+def run_qiskit_circuit(
+    benchmark, circuit, backend, optimization_level, shots, expected_counts, marginalize=0
```

Comment:
Instead of hard coding this as a boolean can we make this optional with `marginalize=None` and then have it take a list of bit indices? That would let people do something like `run_qiskit_circuit(..., marginalize=[0,2,3])` so the result distribution is marginalized on bits 0, 2, and 3.